core/sdk/src/api/firebolt/fb_telemetry.rs

```diff
@@ -192,57 +194,14 @@ impl ExtnPayloadProvider for TelemetryPayload {
     }
 
     fn contract() -> RippleContract {
-        RippleContract::OperationalMetricListener
+        RippleContract::TelemetryEventsListener
     }
 }
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
 pub enum OperationalMetricRequest {
     Subscribe,
     UnSubscribe,
-}
-
-impl ExtnPayloadProvider for OperationalMetricRequest {
```

Comment:
@bobra200 : From POC, I think you moved these to a new module, they now exist elsewhere.